modules/extensions/src/views/lite/components/debugger/views/Inspector.tsx

```diff
@@ -1,54 +1,34 @@
-import { Colors, Icon, Position, Pre, Tooltip } from '@blueprintjs/core'
+import { ContextMenu, Menu, MenuItem, Pre } from '@blueprintjs/core'
+import copy from 'copy-to-clipboard'
 import _ from 'lodash'
 import React, { useState } from 'react'
-import CopyToClipboard from 'react-copy-to-clipboard'
 import JSONTree from 'react-json-tree'
 
 import inspectorTheme from '../inspectorTheme'
 import style from '../style.scss'
 
-const CopyPath = props => {
-  const [copied, setCopied] = useState(false)
-
-  const onCopy = () => {
-    setCopied(true)
-    setTimeout(() => setCopied(false), 1000)
-  }
-
-  return (
-    <Tooltip isOpen={copied} content="Path copied to clipboard" position={Position.TOP}>
-      <CopyToClipboard text={props.path} onCopy={onCopy}>
-        <Icon icon="clipboard" color={Colors.GRAY5} />
-      </CopyToClipboard>
-    </Tooltip>
-  )
-}
-
 interface ExpandedPath {
   path: string
   level: number
 }
 
 export const Inspector = props => {
-  const [expanded, setExpanded] = useState<ExpandedPath[]>([])
-
-  const onContextMenu = (e: React.MouseEvent, path: string, currentLevel: number) => {
+  const handleContextMenu = (e, joinedPaths) => {
     e.preventDefault()
-    path = path.replace('event.', '')
-
-    const entries = [
-      ...expanded.filter(x => x.path !== path),
-      { path, level: (expanded.find(x => x.path === path)?.level ?? currentLevel) + 1 }
-    ]
-
-    setExpanded(_.orderBy(entries, x => x.path.length, ['desc']))
-  }
-
-  const shouldExpand = (key: string[], data, level: number) => {
-    const path = [...key].reverse().join('.')
-    const found = expanded.find(x => path.startsWith(x.path))
 
-    return level <= (found?.level ?? 1)
+    ContextMenu.show(
+      <Menu>
+        <MenuItem
+          text="Copy event path"
+          onClick={() => {
+            console.log('test')
+            copy(`{{${joinedPaths}}}`)
+          }}
+          icon="clipboard"
+        />
```

Comment:
Keep the quick expand feature. When there are a lot of items, it's a pain to open them all manually. Basically, each time you use the feature, it expands all childs one level deeper. You can do it twice to expand all childs two levels, etc.

Reply:
Added it as an option of the context menu for better UX